tests/container-scripts/setup-validators.sh

```diff
@@ -57,11 +58,12 @@ for i in $(seq 1 $NODES);
 do
     cp /genesis.json /validator$i/config/genesis.json
     GAIA_HOME="--home /validator$i"
-    ARGS="$GAIA_HOME --keyring-backend test"
+    ARGS="$GAIA_HOME --keyring-backend test --chain-id=$CHAIN_ID --ip 7.7.7.$i"
+    GENTX_FLAGS="--moniker validator$i --commission-rate 0.05 --commission-max-rate 0.05"
     # the /8 containing 7.7.7.7 is assigned to the DOD and never routable on the public internet
```

Comment:
The Canto antehandler we use has a minimum commission decorator of 5%, so these are also flags which must be provided by our validators for genesis.